ajax/libs/elfinder/package.json

```diff
@@ -24,7 +24,7 @@
       {
         "basePath": "",
         "files": [
-          "js/**/+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)",
+          "js/**/!(*encoding-japanese.min)*.+(html|js|map)",
```

Comment:
Hi @nao-pon, unfortunately, we don't host HTML files as https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md#1-policy-rules-and-guidelines said
> cdnjs does not currently serve HTML or PHP files.

Could you please tell me does a front-end developer need every js file under js directory except extras/encoding-japanese.min.js?